src/main/java/org/jsoup/Jsoup.java

```diff
@@ -238,16 +240,46 @@ public static String clean(String bodyHtml, String baseUri, Whitelist whitelist,
     }
 
     /**
-     Test if the input body HTML has only tags and attributes allowed by the Whitelist. Useful for form validation.
-     <p>The input HTML should still be run through the cleaner to set up enforced attributes, and to tidy the output.
-     <p>Assumes the HTML is a body fragment (i.e. will be used in an existing HTML document body.)
-     @param bodyHtml HTML to test
-     @param whitelist whitelist to test against
-     @return true if no tags or attributes were removed; false otherwise
-     @see #clean(String, org.jsoup.safety.Whitelist) 
+     * Test if the input body HTML has only tags and attributes allowed by the Whitelist. Useful for form validation.
+     * <p>The input HTML should still be run through the cleaner to set up enforced attributes, and to tidy the output.
+     * <p>Assumes the HTML is a body fragment (i.e. will be used in an existing HTML document body.)
+     * @param bodyHtml HTML to test
+     * @param whitelist whitelist to test against
+     * @return true if no tags or attributes were removed; false otherwise
+     * @see #clean(String, org.jsoup.safety.Whitelist)
      */
     public static boolean isValid(String bodyHtml, Whitelist whitelist) {
         return new Cleaner(whitelist).isValidBodyHtml(bodyHtml);
     }
+    /**
+     * Test if the input body HTML has only tags and attributes allowed by the Whitelist. Useful for form validation.
+     * <p>The input HTML should still be run through the cleaner to set up enforced attributes, and to tidy the output.
+     * <p>Assumes the HTML is a body fragment (i.e. will be used in an existing HTML document body.)
+     * @param bodyHtml HTML to test
+     * @param whitelist whitelist to test against
+     * @param baseUri The URL where the HTML was retrieved from. Used to resolve relative URLs to absolute URLs, that occur
+     * before the HTML declares a {@code <base href>} tag.
+     * @return true if no tags or attributes were removed; false otherwise
+     * @see #clean(String, org.jsoup.safety.Whitelist)
+     */
+
+    public static boolean isValid(String bodyHtml, String baseUri,Whitelist whitelist) {
```

Comment:
Why not using newly created method isValidBodyHtml(String bodyHtml,String baseurl)?